WalletWasabi/Tor/TorProcessManager.cs

```diff
@@ -249,6 +249,16 @@ public async Task StopAsync()
 			}
 			Stop?.Dispose();
 			Stop = null;
+
+			if (TorProcess is { } && killTor)
+			{
+				Logger.LogInfo($"Killing Tor process.");
+
+				TorProcess.Kill();
+				using var cts = new CancellationTokenSource(TimeSpan.FromMinutes(1));
+				await TorProcess.WaitForExitAsync(cts.Token, true).ConfigureAwait(false);
```

Comment:
```suggestion
				await TorProcess.WaitForExitAsync(cts.Token, killOnCancel: true).ConfigureAwait(false);
```

Reply:
Also this command may throw an exception too. So likely you may endanger the orderly shutdown of the application if it does, unless it's intended.

Reply:
I would add a logging statement that Tor was successfully closed.

Reply:
ACK!